components/sup/Cargo.toml

```diff
@@ -51,7 +51,7 @@ prost = "*"
 prost-derive = "*"
 prost-types = "0.5.0" # This is current stable, but for some reason gets pulled in as 0.4.0 if we use "*" (2019-03-28)
 rand = "*"
-ratsio = "*"
+ratsio = "^0.2"
```

Comment:
Why add this constraint?

Reply:
because the client library states in its doc that we should make use of 0.2+. i know * should grab the most recent version, but it's not always entirely deterministic, is it?

Reply:
It should synonymous in this case. For good or bad, we've settled on using `*` for everything _unless_ there's a concrete reason to lock to a specific version. In that case, we'll leave a comment as to why we're locking.

(Though now I see the `prost-types` version pin that _I_ added just above... I should revisit that one, but were you seeing something similar?)

Reply:
this wasn't pinned, though i did see other pins and followed that example. i'll remove the pin.